src/backend/test/utils/valid-upgrade-candidates/RC_newerPatch.json

```diff
@@ -0,0 +1,30 @@
+{
+  "currentApkInfo": {
+    "hash": "810ff2fb252a5dee4220f2cb0e6a519891fb67f2f828a6cab4ef8894633b1f50",
+    "size": 8,
+    "versionName": "1.1.0-RC+9a6731e",
+    "minSdkVersion": 1,
+    "applicationId": "com.example",
+    "versionCode": 1,
+    "hashType": "sha256",
+    "platform": "android",
+    "arch": ["arm64-v8a"],
+    "filepath": "non-existent.apk"
+  },
+  "deviceInfo": {
+    "supportedAbis": ["arm64-v8a"],
+    "sdkVersion": 1
+  },
+  "installer": {
+    "hash": "810ff2fb242a5dee4220f2cb0e6a519891fb67f2f828a6cab4ef8894633b1f50",
+    "size": 8,
+    "versionName": "1.1.1-RC+19d986e",
```

Comment:
This is not a valid upgrade candidate. See https://github.com/digidem/mapeo-mobile/issues/578

> **Release Candidates**: Should update to the next RC that shares the same `{major}.{minor}.{patch}`, and any production release that is semver => `{major}.{minor}.{patch}` of the RC.

E.g. RC version should only update to another RC version if the main semver component is equal.
This RC does not share the same `{major}.{minor}.{patch}`, so it should not update. However an RC should update to a non-RC that is semver greater.

Reply:
> E.g. RC version should only update to another RC version if the main semver component is equal.

So `1.0.0-RC+9a6731e` should upgrade to `1.0.0-RC+19d986e`, but how does it know it's a newer RC?

Reply:
> The versionCode / build number should be used to determine updates if the two RC releases have the same version (but different +{sha7}). The higher build number "wins".

How do we determine higher build number?